easybuild/easyconfigs/j/jupyterlab/jupyterlab-0.32.2-CrayGNU-18.08.eb

```diff
@@ -0,0 +1,75 @@
+# @author: robinson, sarafael 
+
+easyblock = 'Bundle'
+
+name = 'jupyterlab'
+version = '0.35.2'
+
+py_maj_ver = '3'
+py_min_ver = '6'
+py_rev_ver = '5.1'
+
+pyver = '%s.%s.%s' % (py_maj_ver, py_min_ver, py_rev_ver)
+pyshortver = '%s.%s' % (py_maj_ver, py_min_ver)
+
+homepage = 'https://github.com/jupyterlab/jupyterlab'
+description = "An extensible environment for interactive and reproducible computing, based on the Jupyter Notebook and Architecture."
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'pic': True, 'verbose': False}
+
+dependencies = [
+    ('jupyter', '1.0.0'),
+    ('jupyterhub', '0.9.4')
+]
+
+# bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+exts_list = [
+    (name, version, {
+        'req_py_majver': '3',
+        'req_py_minver': '6',
+        'use_pip': True,
```

Comment:
What about adding here
 `'installopts': ' --install-option=--skip-npm ',`
to avoid the `npm` install, so this part only does the package installation and the extensions are done later on the postinstall. See [here](https://github.com/jupyterlab/jupyterlab/blob/a2292fe28ea006a0ec0fde34b4aa2959628adf5d/setup.py#L87) and [here](https://github.com/jupyterlab/jupyterlab/blob/a2292fe28ea006a0ec0fde34b4aa2959628adf5d/setupbase.py#L66).